src/rgw/driver/sfs/sqlite/lifecycle/lifecycle_definitions.h

```diff
@@ -0,0 +1,37 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t
+// vim: ts=8 sw=2 smarttab ft=cpp
+/*
+ * Ceph - scalable distributed file system
+ * SFS SAL implementation
+ *
+ * Copyright (C) 2023 SUSE LLC
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ */
+#pragma once
+
+#include "rgw_common.h"
+
+#include <string>
+
+namespace rgw::sal::sfs::sqlite  {
+
+using BLOB = std::vector<char>;
+
+struct DBOPLCHead {
+  std::string lc_index;  // primary key
+  std::string marker;
+  long start_date;
+};
+
+struct DBOPLEntry {
```

Comment:
`s/DBOPLEntry/DBOPLCEntry/`? (note the missing `C` in the former)

Reply:
Fixed